src/internal/pachconfig/config.go

```diff
@@ -139,6 +139,9 @@ type PachdSpecificConfiguration struct {
 	// the proxy, and ProxyTLS for whether or not to use https:// for generated URLs.
 	ProxyHost string `env:"PACHYDERM_PUBLIC_HOST,default="`
 	ProxyTLS  bool   `env:"PACHYDERM_PUBLIC_TLS,default=false"`
+	// Determined integration configuration
```

Comment:
I talk about this below, but why not inject the fields from the secret with secretKeyRef in the manifest.  That way we don't need to talk to k8s to get the username/password.